### PR TITLE
feat: add a new plugin rundown_api

### PR DESCRIPTION
**Pull request by LinkW77**
# Pull Request

## PR Description

Add a new plugin rundown_api

## Type of Change

- [ ] Bug fix
- [x] New feature
- [ ] Performance enhancement
- [ ] Code refactor
- [ ] Documentation update
- [ ] Other, please describe:

## Checklist

Before submitting this PR, please make sure:

- [x] I have read the [CONTRIBUTING.md](../CONTRIBUTING.md) guidelines.
- [x] I have tested my changes locally to ensure they are effective.
- [x] I have updated the necessary documentation (if applicable).

### CODE REVIEW

**Review by jameszyao — APPROVED** [review]
LGTM

**Review by jameszyao — APPROVED** [review]
LGTM

File: plugin/bundles/rundown_api/plugins/get_sport_teams/plugin.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            "ita1": 15,
+            "uefachamp": 16,
+            "uefaeuro": 17,
+            "fifa": 18,
+            "jpn1": 19,
+            "ipl": 20,
+            "t20": 21,
+        }
+
+        if sport.lower() in sport_dict:
```
Comment (DynamesC) [inline plugin/bundles/rundown_api/plugins/get_sport_teams/plugin.py:37]: add null value handling

File: plugin/bundles/rundown_api/plugins/sports_schedule_search/plugin.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            "uefachamp": 16,
+            "uefaeuro": 17,
+            "fifa": 18,
+            "jpn1": 19,
+            "ipl": 20,
+            "t20": 21,
+        }
+
+        if sport.lower() in sport_dict:
+            sport: int = sport_dict.get(sport.lower())
```
Comment (DynamesC) [inline plugin/bundles/rundown_api/plugins/sports_schedule_search/plugin.py:40]: should include null value handling

**Review by DynamesC — CHANGES_REQUESTED** [review]
Please add null value handling when sport name is invalid

**Pull request closed without merging**